src/electron/process_manager.ts

```diff
@@ -151,6 +151,11 @@ export class TunnelManager {
     }
   }
 
+  public setDebug(debug: boolean) {
```

Comment:
Lets change `setDebug(debug)` to `enableDebug()` or similar for readability. Passing false is pointless, since that's the default. Removing the parameter makes it clearer when you need to use the method.

Reply:
Done in c2bbeb2775e078d4c9b5809a44902f1a12fc791c